conduct/conduct_enforcement.md

```diff
@@ -0,0 +1,202 @@
+useful to facilitate a mediation process. This is only an example: the
+Committee can consider mediation in any case, mindful that the process is meant
+to be strictly voluntary and no party can be pressured to participate. If the
+Committee suggests mediation, it should:
+
+* Find a candidate for mediator.
+* Obtain the agreement of the reporter(s). The reporter(s) have complete
+  freedom to decline the mediation idea, or to propose an alternate mediator.
+* Obtain the agreement of the reported person(s).
+* Settle on the mediator: while parties can propose a different mediator than
+  the suggested candidate, only if common agreement is reached on all terms can
+  the process move forward.
+* Establish a timeline for mediation to complete, ideally within two weeks.
+
+The mediator will engage with all the parties and seek a resolution that is
+satisfactory to all.  Upon completion, the mediator will provide a report
+(vetted by all parties to the process) to the Committee, with recommendations
+on further steps.  The Committee will then evaluate these results (whether
+satisfactory resolution was achieved or not) and decide on any additional
+action deemed necessary.
+
+
+## How the committee will respond to reports
+
+When a report is sent to the committee they will immediately reply to the
+reporter to confirm receipt. This reply must be sent within 24 hours, and the
+group should strive to respond much quicker than that.
+
+See the [Reporting Guidelines](*conduct_reporting_online.md*) for details of
+what reports should contain. If a report doesn't contain enough information, the
+committee will obtain all relevant data before acting. The committee is
+empowered to act on the Steering Council’s behalf in contacting any individuals
+involved to get a more complete account of events.
+
+The committee will then review the incident and determine, to the best of their
+ability:
+
+* What happened.
+* Whether this event constitutes a Code of Conduct violation.
+* Who are the responsible party(ies).
+* Whether this is an ongoing situation, and there is a threat to anyone's
+  physical safety.
+
+This information will be collected in writing, and whenever possible the group's
+deliberations will be recorded and retained (i.e. chat transcripts, email
+discussions, recorded voice conversations, etc).
+
+It is important to retain an archive of all activities of this committee to
+ensure consistency in behavior and provide institutional memory for the
+project.  To assist in this, the default channel of discussion for this
+committee will be a private mailing list accessible to current and future
+members of the committee as well as members of the Steering Council upon
+justified request. Only in exceptional circumstances should the committee
+deliberate outside this list (e.g. if a committee member is the target of a
+report).  In such cases, it should still provide a suitable summary of the
+discussion to the list.
```

Comment:
I think we need to give a little more latitude for the CoC members to have private discussions offline, which are then summarized to the list. The way it is written now is that phone calls, etc would only be used in "exceptional circumstances." In my experience, especially if rapid response is needed, phone calls are a really important part of handling these situations. Part of the reason for this is that some situations being handled by the CoC team *will*, from the start, be really thorny. Expecting the CoC members to resolve complex thorny issues with purely online, written communications is ignoring some of the underlying factors that create CoC situations in online communities: it is really difficult to communicate well about complex, emotional issues, without the full set of verbal and non-verbal communication tools.